dulwich/credentials.py

```diff
@@ -0,0 +1,125 @@
+# credentials.py -- support for git credential helpers
+
+# Copyright (C) 2022 Daniele Trifirò <daniele@iterative.ai>
```

Comment:
@jelmer Please let me know if I need to update this header with your name :slightly_smiling_face:

Reply:
This is fine.